swift/ql/test/library-tests/elements/expr/arithmeticoperation/arithmeticoperation.swift

```diff
@@ -10,4 +10,9 @@ func test(c: Bool, x: Int, y: Int, z: Int) {
 	v = x % y;
 	v = -x;
 	v = +x;
+
+   // arithmetic operations with overflow
+   v = x &+ y;
+   v = x &- y;
+   v = x &* y;
```

Comment:
The formatting looks a bit funny here (tabs vs. spaces?)

Reply:
Fixed.